File: source/code/php/idsite/jwt_for_oauth_req.php
$exchangeIdSiteTokenRequest = new \Stormpath\Oauth\ExchangeIdSiteTokenRequest($jwtResponse);
$auth = new \Stormpath\Oauth\ExchangeIdSiteTokenAuthenticator(app('stormpath.application'));
$result = $auth->authenticate($exchangeIdSiteRequest);

File: source/code/java/idsite/jwt_for_oauth_req.java
// req is an HttpServletRequest
String jwtResponse = req.getParameter("jwtResponse");

IdSiteAuthenticationRequest idSiteAuthenticationRequest = OAuthRequests.IDSITE_AUTHENTICATION_REQUEST
    .builder()
    .setToken(jwtResponse)
    .build();

OAuthGrantRequestAuthenticationResult result = Authenticators.ID_SITE_AUTHENTICATOR
    .forApplication(application)
    .authenticate(idSiteAuthenticationRequest);

AccessToken accessToken = result.getAccessToken();
RefreshToken refreshToken = result.getRefreshToken();
